Tools/autotest/arducopter.py

```diff
@@ -67,6 +61,15 @@ def __init__(self, binary,
         self.sitl = None
         self.hasInit = False
 
+    def mavproxy_options(self):
+        ret = super(AutoTestCopter, self).mavproxy_options()
+        if self.frame != 'heli':
+            ret.append('--quadcopter')
```

Comment:
Does this even do anything? I searched MAVProxy source and couldn't find it being used anywhere.